src/lib/create-config.ts

```diff
@@ -19,6 +19,7 @@ import {
     ICspDomainOptions,
     IClientCachingOption,
     IMetricsRateLimiting,
+    IRateLimiting,
```

Comment:
Irate limiting? 😄

Reply:
indeed, I was quite IRate while writing this :P